components/forms/RewardsForm.tsx

```diff
@@ -0,0 +1,138 @@
+import { Account, calculateFee } from "@cosmjs/stargate";
+import { assert } from "@cosmjs/utils";
+import axios from "axios";
+import { NextRouter, withRouter } from "next/router";
+import { useState } from "react";
+import { useAppContext } from "../../context/AppContext";
+import { checkAddress, exampleValidatorAddress } from "../../lib/displayHelpers";
+import Button from "../inputs/Button";
+import Input from "../inputs/Input";
+import StackableContainer from "../layout/StackableContainer";
+
+interface Props {
+  address: string | null;
```

Comment:
I think it would be good to make explicit which address this is. For this component we can say `delegatorAddress` which is then set to the multisig address that signs the transaction.

Reply:
Also this prop should not be `null`able because otherwise we don't create the transaction correctly.